src/db.c

```diff
@@ -1332,14 +1332,24 @@ void delexCommand(client *c) {
 
         decrRefCount(valueobj);
     } else if (!strcasecmp("ifdeq", condition)) {
+        if (validateHexDigest(c->argv[3]->ptr) != C_OK) {
+            addReplyError(c, "must be exactly 16 hexadecimal characters");
```

Comment:
maybe we should use `DIGEST_HEX_LENGTH` instead of `16` directly

Reply:
changed, and moved the error reply to inside t_string.c